pydockerimagebuild.yml

```diff
@@ -0,0 +1,11 @@
+- name: Making Docker Image from DockerFile
+  hosts: anotherinstance18.04
+  remote_user: ubuntu
+  sudo: yes
```

Comment:
Again sudo?

Reply:
docker-compose require permissions as sudo to execute. But, added ubuntu user to docker group. so, needed sudo.